src/plugins/RedPacket/UI/RedPacket.tsx

```diff
@@ -240,6 +235,32 @@ export function RedPacket(props: { redPacket?: RedPacketRecord }) {
     )
 }
 
+function getDescription(
```

Comment:
@neruthes the red packet status is need i18n?

Reply:
I believe that i18n will be good